routing/routes.go

```diff
@@ -27,6 +27,24 @@ func NewRoutes() (r *Routes) {
 			"/api",
 			handlers.Root,
 		},
+		Route{
+			"FailExample",
+			"GET",
+			"/api/500",
+			handlers.FailExample,
+		},
+		Route{
+			"ServiceList",
+			"GET",
+			"/api/namespace/{id:[a-z]+}/services",
+			handlers.ServiceList,
+		},
+		Route{
+			"ServiceShow",
+			"GET",
+			"/api/namespace/{namespace_id:[a-z]+}/service/{id:[a-z]+}",
```

Comment:
If we want to mimic k8s api we should keep `namespaces` and `services` plural (even when the id is provided).
Example of urls can be found here: https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/